internal/services/recoveryservices/backup_protected_vm_resource.go

```diff
@@ -149,6 +188,7 @@ func resourceRecoveryServicesBackupProtectedVMRead(d *pluginsdk.ResourceData, me
 	if properties := resp.Properties; properties != nil {
 		if vm, ok := properties.AsAzureIaaSComputeVMProtectedItem(); ok {
 			d.Set("source_vm_id", vm.SourceResourceID)
+			d.Set("protection_stopped", vm.ProtectionState == backup.ProtectionStateProtectionStopped)
```

Comment:
why are we exposing this as `provisioning_stopped` rather than `provisioning_status` or similar?

Reply:
My thought for this is there are some types not able to set by users, and the users' request is to stop protection and keep the backups.

Reply:
what is the difference between protection stopped and protection paused? These both seem like valid states for a user to set

Reply:
i still agree with tom in that we should call this `protection_state` rather then protection stopped

Reply:
okay, make sense.